controllers/argocd/util_test.go

```diff
@@ -544,6 +580,71 @@ func TestGetArgoApplicationControllerCommand(t *testing.T) {
 	}
 }
 
+func TestGetArgoApplicationControllerSharding(t *testing.T) {
```

Comment:
I see `Spec.Controller.Sharding` test already exists here: https://github.com/argoproj-labs/argocd-operator/blob/master/controllers/argocd/statefulset_test.go#L347 

@AdamSaleh I believe we both looked at this existing test for sharding in one of our calls. Can you please confirm? I see that the existing test and this test do the same thing just that one resides in `statefulset_test.go` and Julia has written inside `util_test.go`.

Reply:
These are very simmilar, but I still think having the additional one is useful, because it focuses on a smaller function - just `getArgoControllerContainerEnv(cr) ` and takes min and max shards into account as well.

Reply:
I believe everything related to sharding exists, if you want to see dynamic sharding testing, it is written here: https://github.com/argoproj-labs/argocd-operator/blob/master/controllers/argocd/statefulset_test.go#L592

Reply:
Agree. So that's why this new test didn't increase test coverage :) Can I close the PR?